src/Yaapii.Atoms/IO/InputAsBytes.cs

```diff
@@ -70,6 +70,8 @@ public byte[] AsBytes()
                 { }
                 output = baos.ToArray();
             }
+
+            Dispose();
```

Comment:
I dont think we can dispose the stream here. What if a user puts a stream into this class and wants to set the stream position to 0, after he got the bytes?